src/main/scala/mesosphere/mesos/Constraints.scala

```diff
@@ -237,10 +243,10 @@ object Constraints {
       diffAfterKill <= 1 || distributionDifference() > diffAfterKill
     }
 
-    def tasksToKillIterator(without: Map[Instance.Id, Instance]): Iterator[Instance] = {
+    def instancesToKillIterator(without: Map[Instance.Id, Instance]): Iterator[Instance] = {
```

Comment:
not sure. i'm trying to avoid large amounts of refactoring because the focus is on enabling pods vs. fixing all the little things that jump out .. with a few exceptions (like preferring immutable.Seq)